test_suite.py

```diff
@@ -35,21 +35,21 @@ def once_per_this_test_run(*lock_keys):
 @pytest.fixture
 def docker_image(docker_client: docker.DockerClient, once_per_test_suite_run) -> str:
```

Comment:
I thought we could use the fixture factory pattern so that we can specify which docker image to create